packages/fx-core/templates/plugins/resource/aad/auth/bot/README.txt

```diff
@@ -168,7 +168,7 @@ Debug your application
 -------------------------
 You can debug your application by:
 
-1. Right-click your project and select Microsoft 365 Agents Toolkit > Prepare Teams app dependencies
+1. Right-click your project and select Microsoft 365 Agents Toolkit > Prepare app dependencies
```

Comment:
What is this file? I'd like @HuihuiWu-Microsoft  to review, since prepare Teams app dependency is not applicable.

Reply:
No Prepare Teams app command now. Change to Select Microsoft 365 Account
" Right-click your project and select Microsoft 365 Agents Toolkit > Select Microsoft 365 Account"